build/Build.cs

```diff
@@ -88,20 +88,17 @@ class Build : NukeBuild
 
     Target CopyToLocalPackages => _ => _
         .OnlyWhenStatic(() => IsLocalBuild)
-        .DependsOn(Pack)
+        .TriggeredBy(Pack)
```

Comment:
Changes in this file as suggested by @matkoch in [this thread](https://nukebuildnet.slack.com/archives/C01SNUZL50V/p1618312679241400) (because this target is skipped if it's not a local build, it's dependencies get skipped too, so we need to re-jig it around.)